### PR TITLE
Advanced autoscroll & Tests refactor

### PR DESCRIPTION
**Pull request by MathisFederico**

### CODE REVIEW

File: tests/integration/blocks/test_block.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        """ Setup reused variables. """
+        self.window = OCBWindow()
+        self.ocb_widget = OCBWidget()
+        self.subwindow = self.window.mdiArea.addSubWindow(self.ocb_widget)
+        self.subwindow.show()
+
+        self.block1 = OCBBlock(title="Testing block 1")
+        self.block2 = OCBBlock(title="Testing block 2")
```
Comment (vanyle) [inline tests/integration/blocks/test_block.py:32]: block2 is not used. Consider commenting this.

**Comment by vanyle** [comment]
For now, I don't think that moving:
```py
        self.window = OCBWindow()
        self.ocb_widget = OCBWidget()
        self.subwindow = self.window.mdiArea.addSubWindow(self.ocb_widget)
        self.subwindow.show()
```
into a utils function makes sense. It might be needed if the setup gets more complicated.
In this case, We'll have to use inheritance for test classes.

More robusteness for the dragging test will be needed in the future but this is another issue.

**Comment by MathisFederico** [comment]
> For now, I don't think that moving:
> 
> ```python
>         self.window = OCBWindow()
>         self.ocb_widget = OCBWidget()
>         self.subwindow = self.window.mdiArea.addSubWindow(self.ocb_widget)
>         self.subwindow.show()
> ```
> 
> into a utils function makes sense. It might be needed if the setup gets more complicated. In this case, We'll have to use inheritance for test classes.

I don't get it, this code is not in utils, it's in test files directly

**Comment by vanyle** [comment]
For now, I don't think that moving *this code* into a utils function makes sense. (I'm referencing the fact that codacy says this is a duplicate but 4 duplicated lines are not enough to be worth a refactor yet.), so it's fine.

**Review by vanyle — APPROVED** [review]

**Pull request merged**